src/eventSchemas/vue/searchSchemas.ts

```diff
@@ -110,6 +110,10 @@ export async function createSearchSchemasWebView(context: ExtContext, node: Regi
     let webviewResult: Result = 'Succeeded'
 
     try {
+        if (!node.regionCode) {
+            vscode.window.showInformationMessage('Unable to determine regionCode for node')
+            return
+        }
```

Comment:
If I make this change, I run into an error that node.regionCode might be undefined. (This is after changing it from type `string | undefined` to `string`). Is there anyway to avoid this?

Reply:
Make sure both `RegistryItemNode` and `SchemasNode` have `regionCode` typed as 'string', then you won't have this issue. Easiest way to do this is to assign at declaration time, e.g. `public readonly regionCode = this.client.regionCode` would narrow the type.